src/openrct2/platform/shared.c

```diff
@@ -763,7 +765,21 @@ void platform_set_cursor_position(sint32 x, sint32 y)
 
 uint32 platform_get_ticks()
 {
-	return SDL_GetTicks();
+#ifdef _WIN32
+    return GetTickCount();
+#else
+    struct timespec ts;
+    if (clock_gettime(CLOCK_MONOTONIC, &ts) != 0) {
+        log_fatal("clock_gettime failed");
+        exit(-1);
+    }
+    return (uint32)(ts.tv_nsec / 1000000);
+#endif
+}
+
+void platform_sleep(uint32 ms)
+{
+    SDL_Delay(ms);
```

Comment:
For Windows you can call `Sleep(ms`).
I haven't worked out how to do this yet for unix.